1-JavaScript/Week4/homework/ex2-mondaysWorth.js

```diff
@@ -29,16 +29,25 @@ const mondayTasks = [
 
 const hourlyRate = 25;
 
-function computeEarnings(/* TODO parameter(s) go here */) {
+function computeEarnings(array, rate) {
```

Comment:
works perfect, congrats! 

another possible solution would be to do the entire operation inside the for each:

```
function computeEarnings(array, rate) {
  const durations = array.map((x) => x.duration);
  let earnings = 0;
  
  durations.forEach((minutes) => {
    earnings += (minutes / 60) * rate;
  });
  
  return '€' + earnings;
}
```